app/src/main/res/layout/fragment_user.xml

```diff
@@ -1,324 +1,354 @@
+                </LinearLayout>
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal">
 
                     <LinearLayout
                         android:layout_width="0dp"
-                        android:layout_height="match_parent"
-                        android:layout_marginLeft="@dimen/dp_12"
-                        android:layout_marginRight="@dimen/dp_12"
+                        android:layout_height="wrap_content"
                         android:layout_weight="1"
-                        android:orientation="vertical"
-                        android:paddingTop="@dimen/dp_10">
+                        android:gravity="center"
+                        android:onClick="@{() -> viewModel.onCallClick()}"
+                        android:orientation="horizontal"
+                        android:padding="@dimen/dp_8">
+
+                        <ImageView
+                            android:layout_width="@dimen/dp_24"
+                            android:layout_height="@dimen/dp_24"
+                            android:layout_marginEnd="@dimen/dp_20"
+                            android:layout_marginRight="@dimen/dp_20"
+                            android:contentDescription="@null"
+                            android:tint="@color/colorPrimary"
+                            bind:srcCompat="@drawable/ic_phone_receiver_24dp"/>
 
-                        <!--user name-->
                         <TextView
-                            android:id="@+id/textView"
-                            android:layout_width="match_parent"
+                            android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
-                            android:text="@{viewModel.user.name}"
-                            android:textColor="@color/colorPrimary"
-                            android:textSize="@dimen/sp_20"/>
+                            android:gravity="center"
+                            android:text="@string/title_call"/>
 
-                        <!--user phone-->
-                        <TextView
-                            android:layout_width="match_parent"
-                            android:layout_height="wrap_content"
-                            android:text="@{viewModel.user.phone}"
-                            android:textSize="@dimen/sp_15"/>
+                    </LinearLayout>
+
+                    <View
+                        android:layout_width="@dimen/dp_1"
+                        android:layout_height="match_parent"
+                        android:layout_marginLeft="@dimen/dp_5"
+                        android:layout_marginRight="@dimen/dp_5"
+                        android:background="?android:attr/listDivider"/>
+
+                    <LinearLayout
```

Comment:
WARNING: [UseCompoundDrawables] This tag and its children can be replaced by one `<TextView/>` and a compound drawable
 A `LinearLayout` which contains an `ImageView` and a `TextView` can be more efficiently handled as a compound drawable (a single TextView, using the `drawableTop`, `drawableLeft`, `drawableRight` and/or `drawableBottom` attributes to draw one or more images adjacent to the text).

If the two widgets are offset from each other with margins, this can be replaced with a `drawablePadding` attribute.

There's a lint quickfix to perform this conversion in the Eclipse plugin.